carsons/carsons.py

```diff
@@ -246,22 +281,48 @@ def compute_d(self, i, j) -> float:
             # Distance between two neutral/phase conductors
             return distance_ij
 
-    def compute_X(self, i, j) -> float:
-        Q_first_term = super().compute_Q(i, j, 1)
-
-        # Simplify equations and don't compute Dᵢⱼ explicitly
-        kᵢⱼ_Dᵢⱼ_ratio = sqrt(self.ω * self.μ / self.ρ)
-        ΔX = Q_first_term * 2 + log(2)
-
-        if i == j:
-            X_o = -log(self.gmr[i]) - log(kᵢⱼ_Dᵢⱼ_ratio)
-        else:
-            X_o = -log(self.compute_d(i, j)) - log(kᵢⱼ_Dᵢⱼ_ratio)
-
-        return (X_o + ΔX) * self.ω * self.μ / (2 * π)
-
     def GMR_cn(self, phase) -> float:
         GMR_s = self.neutral_strand_gmr[phase]
         k = self.neutral_strand_count[phase]
         R = self.radius[phase]
         return (GMR_s * k * R**(k-1))**(1/k)
+
+
+class MultiConductorCarsonsEquations(ModifiedCarsonsEquations):
+    def __init__(self, model):
+        super().__init__(model)
+        self.radius: Dict[str, float] = model.radius
+        self.insulation_thickness: Dict[str, float] = \
+            model.insulation_thickness
+
+    def compute_d(self, i, j) -> float:
+        # Assumptions:
+        # 1. All conductors in the cable are touching each other and
+        #    therefore equidistant.
+        # 2. In case of quadruplex cables, the space between conductors
+        #    which are diagonally positioned is neglected.
+        return (self.radius[i] + self.radius[j]
+                + self.insulation_thickness[i] + self.insulation_thickness[j])
+
+    def compute_D(self, i, j) -> float:
+        return self.phase_positions[i][1] + self.phase_positions[j][1]
```

Comment:
Ok, last comment i swear! the coverage report suggests this never gets called, which makes sense to me given how modified carsons works. so maybe we can just drop it?

Reply:
haha that's okay. right yeah it makes sense modified carsons don't need it. i'll drop it

Reply:
updated in f3bfad9